.travis.yml

```diff
@@ -0,0 +1,11 @@
+sudo: required
```

Comment:
Not possible actually, there is no travis add-on yet for postgres 9.5 which I use, and need for the upsert: 
`ON CONFLICT UPDATE`